Assets/HotmaxEditorVR/inputManager.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class inputManager : MonoBehaviour
+{
+    // -- controller refs
+    public static Valve.VR.InteractionSystem.Hand hand1;
+    public static Valve.VR.InteractionSystem.Hand hand2;
+    public static SteamVR_TrackedController trackedController1;
+    public static SteamVR_TrackedController trackedController2;
+    private SteamVR_TrackedObject trackedObject2;
+    public static SteamVR_Controller.Device device2;
```

Comment:
can we call device2 something else more descriptive?

Reply:
yeah, I can call it selectorHand or rightHand

Reply:
selectorDevice would be better actually